lib/bolt/bolt_option_parser.rb

```diff
@@ -161,7 +161,7 @@ def get_help_text(subcommand, action = nil)
           bolt
 
       USAGE
-          bolt <subcommand> [action] [options]
+          bolt SUBCOMMAND [ACTION] [OPTIONS]
```

Comment:
I think all-caps makes sense for things like `-t TARGETS` to clarify which part is the flag and which part isn't, but otherwise lowercase is standard for CLI tools. The `<subcommand>` brackets are also important to instruct the user that they need to fill that part in - so `bolt subcommand --help` isn't valid.

- [docopt standard](http://docopt.org/)
- [Microsoft CLI syntax](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/command-line-syntax-key)

Reply:
I'll switch back to using brackets. Fwiw, this is the doc I had been referencing, which suggested all caps: https://developers.google.com/style/code-syntax . Everyone's gotta have their own standard. :P